ADOPTERS.md

```diff
@@ -0,0 +1,29 @@
+# Metal3.io Adopters
+
+| Type | Name | Since | Website | Use-Case |
+|:-|:-|:-|:-|:-|
+| Integration | Ericsson | 2019 | [link](https://www.ericsson.com/en/portfolio/digital-services/cloud-infrastructure/cloud-container-distribution)
+| As a Kubernetes distributor we are building Cloud Container Distribution (CCD)
+and integrating metal3 project for baremetal deployments and for baremetal
```

Comment:
```suggestion
and integrating Metal3 project for baremetal deployments and for baremetal
```

Reply:
This came from the adoprters directly , so perhaps its fine, not so strong opinion on this. I am ok as it is since we have both here and there.